bodhi/tests/server/scripts/test_initializedb.py

```diff
@@ -19,12 +19,12 @@
 
 from io import StringIO
 from unittest import mock
-import unittest
+import pytest
```

Comment:
```suggestion

import pytest
```

Reply:
pytest is a third party module, so we need a newline between it and the standard modules, otherwise it will fail the style tests.